sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -28,6 +28,7 @@
 enz = Symbol('enz', extended_nonzero=True)
 a = Symbol('a', algebraic=True)
 na = Symbol('na', nonzero=True, algebraic=True)
+nb = Symbol('nb', integer=True)
```

Comment:
Why can't you use one of the integer=True symbols already defined above?

Reply:
Yeah, I will correct this,